### PR TITLE
Flightmodes - split takeoff

### PR DESCRIPTION
**Pull request by hamishwillee**
This splits takeoff mode into FW and MC variants. Links have been fixed up to each doc, and also the original is a redirect.

This was a bit trickly because we referenced the FW takeoff as place that is also used for mission fw takeoff and we also mention in basic flying. I'll add some comments inline but I am fairly confident of this.

@sfuhrer I thought I might hold off merging this one, just to give you a chance to review. However if I see no comment I probably will merge next week.

### CODE REVIEW

File: en/flight_modes/mission.md [hunk 1]
```diff
@@ -215,23 +215,25 @@ If the vehicle is already flying when the mission is started, a takeoff mission
 
 Starting flights with mission takeoff (and landing using a mission landing) is the recommended way of operating a plane autonomously.
 
-:::note
```
Comment (hamishwillee) [inline en/flight_modes/mission.md:218]: This page updated. You are still expected to go to the Takeoff mode page to understand config of the catapult or runway BUT you don't need to go there to understand how the mission item works.

File: en/flight_modes_fw/takeoff.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
+# Takeoff Mode (Fixed-wing)
+
+[<img src="../../assets/site/position_fixed.svg" title="Position fix required (e.g. GPS)" width="30px" />](../getting_started/flight_modes.md#key_position_fixed)
```
Comment (hamishwillee) [inline en/flight_modes_fw/takeoff.md:3]: Restructured but content largely the same. The main difference is that this doesn't really refer to missions much at all.

File: en/flight_modes/mission.md (hunk 1 above)
Comment (hamishwillee) [reply en/flight_modes/mission.md:218]: Note, I hope to split this into MC and FW pages as well. Will need help to understand what an FW mission looks like in terms of supported mission items etc.

File: en/flying/basic_flying.md [hunk 3]
```diff
@@ -55,6 +53,32 @@ The vehicle may disarm if you take too long to take off after arming (tune the t
...
+
+1. Point the vehicle in the direction you want it to take off, in order to set the course (ideally facing the wind)
+1. Activate [Takeoff mode (FW)](../flight_modes_fw/takeoff.md)
+1. [Arm the vehicle](#arm) using an [RC switch](../config/flight_mode.md) or ground station
+1. Throw/launch the vehicle in the direction you want it to take off.
+
+   Motors will start after the launch is detected, after which the behaviour is the same as for runway takeoff.
+
+For more information see [Takeoff mode (FW)](../flight_modes_fw/takeoff.md).
+Using a takeoff item defined in [a mission plan](../flight_modes/mission.md#fw-mission-takeoff) is also recommended.
```
Comment (hamishwillee) [inline en/flying/basic_flying.md:80]: The manual flying section was very MC specific. I've started building it out with FW stuff as well. My plan is to split this too and put it into MC and FW sections.

I might have put too much here on takeoff, or too little. Thoughts?

File: en/flight_modes/mission.md (hunk 1 above)
Comment (sfuhrer) [reply en/flight_modes/mission.md:218]: Yes Mission should be split up as well, the behavior is greatly different between MC and FW

File: en/flying/basic_flying.md [hunk 4]
```diff
@@ -37,16 +37,14 @@ A VTOL vehicle can only arm in multicopter mode (by default - arming in fixed-wi
...
 ## Takeoff
 
-The easiest way to takeoff (after [arming the vehicle](#arm)) is to use the automatic [Takeoff mode](../flight_modes/takeoff.md).
+### Multicopter takeoff
+
+The easiest way to takeoff (after [arming the vehicle](#arm)) is to use the automatic [Takeoff mode (MC)](../flight_modes_mc/takeoff.md).
 Usually this is triggered from an [RC switch](../config/flight_mode.md) or ground station.
 
-Multicopter (and VTOL in multicopter mode) pilots can take off *manually* by enabling [position mode](../flight_modes/README.md#position_mc), arming the vehicle, and then raising the throttle stick above 62.5%.
+Multicopter (and VTOL in multicopter mode) pilots can also take off *manually* by enabling [Position mode (MC)](../flight_modes/README.md#position_mc), arming the vehicle, and then raising the throttle stick above 62.5%.
```
Comment (sfuhrer) [inline en/flying/basic_flying.md:45]: They can takeoff in any manual mode. 
How about:

Comment (sfuhrer) [inline en/flying/basic_flying.md:45]:
```suggestion
Multicopter (and VTOL in multicopter mode) pilots can take off *manually* by enabling any manual mode, arming the vehicle and then raising the throttle stick until the motors produce enough thrust to leave the ground.
In  [Position mode (MC)](../flight_modes/README.md#position_mc) or [Altitude mode (MC)](../flight_modes/README.md#altitude_mc) the altitude is controlled by the controller, and the throttle stick has to be increased to above 62.5% to make the vehicle leave the ground. 

```

File: en/flying/basic_flying.md [hunk 5]
```diff
@@ -37,16 +37,14 @@ A VTOL vehicle can only arm in multicopter mode (by default - arming in fixed-wi
 
 ## Takeoff
 
-The easiest way to takeoff (after [arming the vehicle](#arm)) is to use the automatic [Takeoff mode](../flight_modes/takeoff.md).
+### Multicopter takeoff
+
+The easiest way to takeoff (after [arming the vehicle](#arm)) is to use the automatic [Takeoff mode (MC)](../flight_modes_mc/takeoff.md).
```
Comment (sfuhrer) [inline en/flying/basic_flying.md:42]: I wouldn't call this the easiest way to takeoff with a MC. TBH I would completely remove the reference here (it's all about manual flying), or max have a reference further down that takeoff could also be automatic and then the user can take over.
Similar story for FW.

File: en/flight_modes/mission.md (hunk 1 above)
Comment (hamishwillee) [reply en/flight_modes/mission.md:218]: Do you think you could outline the differences - both what is different in the current doc and "overall"?

File: en/getting_started/flight_modes.md [hunk 6]
```diff
@@ -364,7 +364,7 @@ The PX4 GCS is called [QGroundControl](https://docs.qgroundcontrol.com/master/en
 
 [<img src="../../assets/site/automatic_mode.svg" title="Automatic mode" width="30px" />](#key_automatic)&nbsp;
 
-[Takeoff mode](../flight_modes/takeoff.md#fixed_wing) initiates the vehicle takeoff sequence.
+[Takeoff mode](../flight_modes_fw/takeoff.md#fixed_wing) initiates the vehicle takeoff sequence.
```
Comment (hamishwillee) [inline en/getting_started/flight_modes.md:367]:
```suggestion
[Takeoff mode](../flight_modes_fw/takeoff.md) initiates the vehicle takeoff sequence.
```

File: en/flying/basic_flying.md [hunk 7]
```diff
@@ -55,6 +53,32 @@ The vehicle may disarm if you take too long to take off after arming (tune the t
 The [Failure Detector](../config/safety.md#failure-detector) will automatically stop the engines if there is a problem on takeoff.
 :::
 
+### Fixed-wing Takeoff
+
+Automatic takeoff using [Takeoff mode (FW)](../flight_modes_fw/takeoff.md) is highly recommended for fixed-wing vehicles!
```
Comment (sfuhrer) [inline en/flying/basic_flying.md:58]: I would link the auto takeoff, but recommend that manually piloted flights are also started manually. If the pilot is not capable of a safe takeoff then the landing, which usually is harder, is likely not going to succeed either. Bette then to do the whole flight in a "auto" mindset, with mission takeoff and mission landing.

File: en/flying/basic_flying.md (hunk 3 above)
Comment (sfuhrer) [reply en/flying/basic_flying.md:80]: too much, see comments above.

**Pull request merged**